scripts/common/rook.sh

```diff
@@ -432,3 +432,47 @@ function add_rook_store_object_migration_status() {
     kubectl patch configmap kurl-migration-from-rook -n kurl --type merge -p '{"data":{"DID_MIGRATE_ROOK_OBJECT_STORE":"1"}}'
     export DID_MIGRATE_ROOK_OBJECT_STORE=1
 }
+
+# rook_maybe_migrate_from_openebs may migrate data from OpenEBS to Rook when all the following
+# conditions are met:
+# - Ekco version is >= 0.27.1.
+# - OpenEBS and Rook are selected on the Installer.
+# - Rook's minimum node count is set to a value > 1.
+# - The number of nodes on the cluster is >= than the Rook minimum node count.
+# - The 'scaling' storage class exists.
+function rook_maybe_migrate_from_openebs() {
```

Comment:
@ricardomaraschini I'm curious why you decided to put this function here rather than in the Rook add-on code?

Reply:
Because we have rook.sh whcih has the common functions for Rook. Therefore, I think that is the reason.

Reply:
Because this isn't run as part of the rook addon, it's run as part of the join/upgrade code